packages/components/psammead-inline-link/src/__snapshots__/index.test.jsx.snap

```diff
@@ -9,8 +9,7 @@ exports[`InlineLink should render correctly 1`] = `
 }
 
 .c0:visited {
-  color: #757575;
-  border-bottom: 1px solid #757575;
+  border-bottom: 1px solid;
```

Comment:
the colour completely disappears here. Is this expected? Shouldn't it use whatever the value of `C_METAL` is?